### PR TITLE
pympress: 1.7.0 -> 1.7.2

### PR DESCRIPTION
**Pull request by drupol**
Hello,

I'm trying to use this software in NixOS and I notice that it is broken, see https://github.com/NixOS/nixpkgs/pull/147046#issuecomment-1238575502

~~In the meantime, I'm providing this bump which is also broken.~~ Thanks @Minion3665 who fixed the issue on Matrix.

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by Minion3665 — CHANGES_REQUESTED** [review]
Looks good to me except for the commit message, which will need to be changed to fit contributing.md. I'm going to do that now I think

**Review by Minion3665 — APPROVED** [review]
And with that, this looks good to me

File: pkgs/applications/office/pympress/default.nix [hunk 1]
```diff
@@ -6,19 +6,37 @@
...
+        gtk3
+        pango
+        harfbuzz
+        gdk-pixbuf
+        atk
+      ])}:${lib.makeSearchPathOutput "out"
+      "lib/girepository-1.0" ([
+        poppler_gi
+      ])}"
+  ];
```
Comment (SuperSandro2000) [inline pkgs/applications/office/pympress/default.nix:38]: This should all be done automatically if gobject-introspection is moved to nativeBuildInputs

Comment (Minion3665) [reply pkgs/applications/office/pympress/default.nix]: Good catch, thank you! Pol is away right now, so I've fixed that. It appears to work fine with your suggested change

Comment (SuperSandro2000) [reply pkgs/applications/office/pympress/default.nix]: You are sure we do not need to add the other packages to buildInputs? If yes then please remove them from inputs again.

Comment (Minion3665) [reply pkgs/applications/office/pympress/default.nix]: > You are sure we do not need to add the other packages to buildInputs? If yes then please remove them from inputs again.

It seems to be working fine on my machine without adding the other packages to buildInputs, although this does seem to contradict with some of the errors we were getting earlier while making up the `makeWrapperArgs`. Running `nix why-depends` does seem to show them as dependencies though, so I suspect that this is why gobject-introspection can still pick them up

(e.g. for atk)
```
/nix/store/976jflnzg5ias7l0r91z4m70w645d6lg-pympress-1.7.2
├───/nix/store/gnqjdvvy484fnhrj4cfwkq9mamkgf6mp-atk-2.38.0
└───/nix/store/2bs64xwzj41afvk5h7hzma6dp0sx83sx-gtk+3-3.24.34
    ├───/nix/store/gnqjdvvy484fnhrj4cfwkq9mamkgf6mp-atk-2.38.0
    └───/nix/store/6vz3f3wzvcbq1p9z6lvl0znispi7xz00-at-spi2-atk-2.38.0
        └───/nix/store/gnqjdvvy484fnhrj4cfwkq9mamkgf6mp-atk-2.38.0```

**Comment by drupol** [comment]
Thanks for fixing the issue while I was sleeping !

**Comment by drupol** [comment]
I just tested the new commits and everything is ok now. Let's get this thing merged :)

**Pull request merged**